server/controller/accountsMangment/customer/getCustomers.js

```diff
@@ -0,0 +1,12 @@
+const { getCustomers } = require('../../../database/queries/customer/getCustomers');
+
+const getCustomer = (request, response) => {
+  getCustomers()
+    .then((res) => {
+      if (res.rows) { response.status(200).json(res.rows); }
```

Comment:
we agreed to send success data as `{result: the result will be send}`
then the response will send as like `response.status(200).send({result: res.rows});`

Reply:
ok